File: 2019/20190703 - fizzbuzz - ruby/problem_spec.rb
require_relative 'problem'

describe "fizzbuzz" do
  it "with 1" do
    expect(fizzbuzz(1)).to eq(1)
  end

  it "with 2" do
    expect(fizzbuzz(2)).to eq(2)
  end

  it "with 3" do
    expect(fizzbuzz(3)).to eq("fizz")
  end
  it "with 4" do
    expect(fizzbuzz(4)).to eq(4)
  end
end

File: 2020/20200108 - fizzbuzz - javascript/problem_spec.js
var assert  = require('assert'),
    fizzBuzz = require('./problem').fizzBuzz;

describe('FizzBuzz', function() {
    it('1 is 1', function () {
        assert.equal(fizzBuzz(1), 1)
    })

    it('2 is 2', function () {
        assert.equal(fizzBuzz(2), 2)
    })

    it('3 is Fizz', function () {
        assert.equal(fizzBuzz(3), 'Fizz')
    })

    it('4 is 4', function () {
        assert.equal(fizzBuzz(4), 4)
    })
})